source/Playnite/Emulators/FightCade.yaml

```diff
@@ -5,4 +5,4 @@ Profiles:
     DefaultArguments: '"{ImageNameNoExt}"'
     Platforms: [Various]
     ImageExtensions: [zip, bin]
-    ExecutableLookup: ^ggpofba\.exe$

```

Comment:
Did this permanently change in the app? Won't this prevent use of older versions?